src/guides/v2.3/config-guide/deployment/pipeline/build-system.md

```diff
@@ -87,6 +87,11 @@ To set up the build system:
    ```
 
 1. Save your changes to `.gitignore` and exit the text editor.
+1. The build system should use either [default mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html#default-mode) or [developer mode]({{ page.baseurl }}/config-guide/bootstrap/magento-modes.html#developer-mode):
+
+   ```bash
+   bin/magento deploy:mode:set default or developer
```

Comment:
Also, please modify the command like this.
```
bin/magento deploy:mode:set {mode}
```
**{mode}**  is required. It can be either `default` or `developer`

Reply:
Hello @ajithkumar-maragathavel,

Thanks for your review and suggestion. The requested changes are done. Kindly verify it.

Thanks.